.pre-commit-config.yaml

```diff
@@ -269,6 +269,7 @@ repos:
          # so the user is informed to take action.
          - --line-length
          - '99999'
+         - --no-transform-percent
```

Comment:
I had to add this because the pre-commit checks were enforcing percent strings, while this listing tool was transforming them into f-strings automatically. Hopefully, that's okay.

Reply:
- https://github.com/apache/airflow/pull/36780#discussion_r1452011362
- https://github.com/apache/airflow/pull/36780#discussion_r1452012713